src/createManager.js

```diff
@@ -72,7 +72,7 @@ const protoManager = {
 
   openMenu(openOptions) {
     if (this.isOpen) return;
-    openOptions = openOptions || {};
+    openOptions = openOptions || { focusMenu: true };
```

Comment:
That would definitely work - and it's more in line with the rest of the code. Thanks! I'll try to look into this and the test/toggle function this week. Would be great to have my branch back into your repo/the npm module!